types/react-big-calendar/lib/addons/dragAndDrop.d.ts

```diff
@@ -0,0 +1,14 @@
+import BigCalendar, { BigCalendarProps, Event, stringOrDate } from '../../index';
+import React = require('react');
```

Comment:
Should use esmodule imports here

Reply:
I changed it because the build failed with an ES import. I just changed it back so I'll see what the build says.

Reply:
I think you need `import * as React from 'react';` because esModuleInterop is disabled

Reply:
Okay that worked, thanks!